projects/igniteui-angular/src/lib/grids/hierarchical-grid/hierarchical-grid-navigation.service.ts

```diff
@@ -1,6 +1,7 @@
 import { IgxGridNavigationService } from '../grid-navigation.service';
 import { IgxHierarchicalGridComponent } from './hierarchical-grid.component';
 import { first } from 'rxjs/operators';
+import { IgxColumnComponent, FilterMode } from '../grid';
```

Comment:
Wrong import! Either import them from the respective files or explicitly import from the barrel file